src/app/auth/shared/auth.service.ts

```diff
@@ -1,19 +1,22 @@
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { Observable } from 'rxjs';
+import { environment } from '../../../environments/environment';
```

Comment:
you can import this as
`import { environment } from 'src/environments/environment';`